pkg/asset/installconfig/installconfig.go

```diff
@@ -165,3 +164,21 @@ func (a *InstallConfig) Load(f asset.FileFetcher) (found bool, err error) {
 	a.File, a.Config = file, config
 	return true, nil
 }
+
+func fetchInstallConfigFile(f asset.FileFetcher) (*asset.File, error) {
+	names := []string{installConfigFilename, deprecatedInstallConfigFilename}
+	for i, name := range names {
+		file, err := f.FetchByName(name)
```

Comment:
WDYT about using https://github.com/openshift/installer/blob/master/pkg/asset/filefetcher.go#L16
to get `install-config.*` and then using the extension to decode the first/last file?
`yml,yaml` -> yaml.Unmarshal
`json` - > json.Unmarshal

Reply:
Do we want to support all versions going forward? I was taking the approach that `install-config.yml` would be deprecated and removed soon. I personally don't see a need to support multiple different extensions. Even if the end user wants to use json instead of yaml, they could still use the yaml extensions, and the installer could still unmarshal it as yaml (since json is yaml).

Reply:
:+1: